problem1.py

```diff
@@ -1 +1,21 @@
 
+# title
+print ("****Fine based on how much over the speed limit the driver is going.****")
+
+# get speed limit and user's speed
+speed_limit = int(input("Speed limit (in km/hr): "))
+user_speed = int(input("Enter the recorded speed of the car (in km/hr): "))
+
+# determine fine based on the speed limit and user's speed
+if speed_limit >= user_speed:
+  print ("Congratulations, you are within the speed limit! ")
+  exit()
```

Comment:
no need to put `exit()` since there is no other code to run after the if statement structure